install.sh

```diff
@@ -3222,17 +2422,18 @@ if [ "$INSTALL_ETC" = "yes" ] ; then
    done
 fi
 
-# --- Set ownership or remove OpenNebula directories ---
+# --- Set ownership, remove OpenNebula directories or delete other arch files---
 
 if [ "$UNINSTALL" = "no" ] ; then
-    if [ "$SUNSTONE" = "yes" ] || [ "$SUNSTONE_DEV" = "yes" ] ; then
-        touch $DESTDIR$VAR_LOCATION/sunstone/main.js
-        rm -f $DESTDIR$SUNSTONE_LOCATION/public/dist/main.js
-        ln -s $VAR_LOCATION/sunstone/main.js $DESTDIR$SUNSTONE_LOCATION/public/dist/main.js
-    fi
     for d in $CHOWN_DIRS; do
         chown -R $ONEADMIN_USER:$ONEADMIN_GROUP $DESTDIR$d
     done
+
+	if [ $ARCH = 'x86_64' ]; then
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Double quote to prevent globbing and word splitting.